angular_token_auth.js

```diff
@@ -12,6 +12,7 @@
         LOGOUT_REDIRECT_URL: '/logout/',
         AUTH_HEADER_PREFIX: 'Token',
         ALLOWED_HOSTS: [],
+        STORAGE_METHOD: 'auto',
```

Comment:
I think it's use is the same if you were to set it to `null` or any other value than `'cookie'` or `'localstorage'`. In that case, it's more a cue to developers than it is functional.